### PR TITLE
[ENGOPS-1530] - Conversion of cpf to maven

### PR DESCRIPTION
**Pull request by diogofscmariano**
@pamval pr for cpf
There are changes in xml files that are not being handled correctly by git. That is why many files are marked as removed and moved.

### CODE REVIEW

**Comment by pamval** [comment]
this is working fine locally for me.

@tgf, please review and merge if appropriate.

File: core/src/main/java/pt/webdetails/cpf/packager/origin/RepositoryOrigin.java [hunk 1]
```diff
@@ -20,7 +20,7 @@ public RepositoryOrigin(String basePath) {
 
   @Override
   public IReadAccess getReader(IContentAccessFactory factory) {
-    return factory.getPluginRepositoryReader(RepositoryHelper.appendPath("..", basePath));
+    return factory.getPluginRepositoryReader(RepositoryHelper.appendPath( "src/main", basePath));
```
Comment (tgf) [inline core/src/main/java/pt/webdetails/cpf/packager/origin/RepositoryOrigin.java]: @diogofscmariano this doesn't look right

**Comment by tgf** [comment]
@diogofscmariano a few '..' -> 'src/main outside test classes, please review

**Comment by diogofscmariano** [comment]
pr updated

**Pull request merged**